spec/lib/twingly/url_spec.rb

```diff
@@ -102,6 +114,16 @@ def valid_urls
       end
     end
 
+    context "IDNA 2003 vs IDNA 2008" do
```

Comment:
> Now I just used context as a "title" and then added an sentence that would read well (IMHO) in the spec output:

What makes sense to me, when it comes to context, is that you are describing a setting. It's not obvious in a library like this, so I agree it's hard too. It would be more obvious if we dealt with higher level stuff like "when logged in" vs "when not logged in".